pyplanet/apps/contrib/mx/__init__.py

```diff
@@ -32,14 +33,52 @@ def __init__(self, *args, **kwargs):
 		await self.api.create_session()
 
 	async def on_start(self):
-		await self.instance.permission_manager.register('add_remote', 'Add map from remote source (such as MX)', app=self, min_level=2)
+		await self.instance.permission_manager.register('add_remote', 'Add map from remote source (such as MX)',
```

Comment:
Please use a return after the parentheses to convert to multi-line parameters (readability).

Reply:
ok